server/app.js

```diff
@@ -9,29 +10,34 @@ app.use(express.json());
 
 app.get('/api/similarProducts/products/:id', (request, response) => {
   console.log(request.params);
-  pg.getOne(request, response);
+  //pg.getOne(request, response);

```

Comment:
Big fan of the way that your server and API calls are separated by file. This is something I would definitely do if I had more time, because it looks so much nicer.